packages/reports/src/Components/Routes/route-with-sub-routes.tsx

```diff
@@ -1,10 +1,29 @@
 import React from 'react';
 import { Redirect, Route } from 'react-router-dom';
+import { RouteComponentProps } from 'react-router';
 import { redirectToLogin, isEmptyObject, routes, removeBranchName, default_title } from '@deriv/shared';
 import { getLanguage } from '@deriv/translations';
+import { TRoute } from 'Constants/routes-config';
+import type { TStores } from '@deriv/stores';
 
-const RouteWithSubRoutes = route => {
-    const renderFactory = props => {
+// this type already described in ../../Containers/routes.tsx, but previous PR is not merged yet
+// TODO: export this type during refactor
+type TPassthrough = {
+    root_store: TStores;
+    WS: Record<string, any>;
+};
+// the same thing, export type from ../binary-router when previous PR will be merged
```

Comment:
I think you may remove all of these comments

Reply:
> I think you may remove all of these comments

Ok, I'll delete. What about types? Do I need to create separate file for types or leave it for now?

Reply:
You just need to wait for the types to be merged right? You can move the card to `Blocked` and set the other card as dependency first.

Reply:
> You just need to wait for the types to be merged right? You can move the card to `Blocked` and set the other card as dependency first.

It's just for one type. Also, I think we will need a common file with types for the next migrations, would be better to create it earlier than later) What do you think, can I try to create it for this PR?

Reply:
Ah I see, yes, create a common type here. It's good to make it earlier as you mentioned👍🏻